### ISSUE TITLE
Input type password should allow aria-required

### ISSUE THREAD

**Issue by cpandhi** [w3c/html-aria#76]
As per the latest draft of the document conformance for use os aria in HTML table, input type=password does not have an implied role and hence can use only global properties and the aria-required is not a gobal property. 
But according to the ARIA1.1 specs,
Use the aria-required attribute on any element that is allowed the required attribute in HTML5. must not be set to falseif the required attribute is set. - (changed)
may also be used for elements that have an ARIA role which allows the aria-required.

So is that a bug in the spec or in the table?
